README.md

```diff
@@ -65,16 +65,16 @@ The broken constraints are tracked, and the information is passed to the fronten
 
 The table of error codes and their description:
 
-|Constraints                    |Code|Other keys                                                     |
-|-------------------------------|:--:|---------------------------------------------------------------|
-|Always at least one nurse      |AON | day::Int, segments::(Vector{[segment_begin, segment_end]})    |
-|Workers number during daytime  |WND | day::Int, required::Int, actual::Int                          |
-|Workers number during night    |WNN | day::Int, required::Int, actual::Int                          |
-|Disallowed shift sequence      |DSS | day::Int, worker::String, preceding::Shift, succeeding::Shift |
-|Lacking long break             |LLB | week::Int, worker::String                                     |
-|Worker undertime hours         |WUH | hours::Int, worker::String                                    |
-|Worker overtime hours          |WOH | hours::Int, worker::String                                    |
-|Multiple teams simultaneously  |MWT | day::Int, from::Int, to::Int, teams::Vector{teams_names}      |
+|Constraints                    |Code|Other keys                                                                            |
+|-------------------------------|:--:|--------------------------------------------------------------------------------------|
+|Always at least one nurse      |AON | day::Int, segments::Vector{[segment_begin, segment_end]}                             |
+|Workers number during daytime  |WND | day::Int, required::Int, actual::Int                                                 |
+|Workers number during night    |WNN | day::Int, required::Int, actual::Int                                                 |
+|Disallowed shift sequence      |DSS | day::Int, worker::String, preceding::Shift, succeeding::Shift                        |
+|Lacking long break             |LLB | week::Int, worker::String                                                            |
+|Worker undertime hours         |WUH | hours::Int, worker::String                                                           |
+|Worker overtime hours          |WOH | hours::Int, worker::String                                                           |
+|Worker teams collision         |WTC | day::Int, segments::Vector{[segment_begin, segment_end]}, teams::Vector{String}      |
```

Comment:
A gdzie info o pracownikach? Chyba doszliśmy do wniosku, że nie potrzebujemy infa o teamach, wystarczą pracownicy, a front sobie ogarnie jak to wyświtlać, i których teamów to dotyczy?